include/veriblock/blockchain/vbk_block_addon.hpp

```diff
@@ -27,6 +27,12 @@ struct VbkBlockAddon :
 
   bool payloadsIdsEmpty() const { return vtbids.empty(); }
 
+  template <typename pop_t, typename pop_id_t>
+  std::vector<pop_id_t>& getPayloadIds();
```

Comment:
Why is it so important to specify this method with the pop_t type ?  Why does not enough only pop_id_t ?